pages/docs/tutorials/websocket/websocket-request-reply.md

```diff
@@ -0,0 +1,369 @@
+
+<Remember>
+The sample request and response payloads are extracted from <a href="https://api.slack.com/apis/connections/socket#events">Slack's official documentation</a>.
+</Remember>
+
+<CodeBlock language="yaml">
+{`  schemas:
+    hello:
+      type: object
+      properties:
+        type:
+          type: string
+          description: A hello string confirming WebSocket connection
+        connection_info:
+          type: object
+          properties:
+            app_id:
+              type: string
+        num_connections:
+          type: integer
+        debug_info:
+          type: object
+          properties:
+            host:
+              type: string
+            started:
+              type: string
+            build_number:
+              type: integer
+            approximate_connection_time:
+              type: integer
+    reaction:
+      type: object
+      properties:
+        envelope_id:
+          type: string
+          description: 'Unique ID assigned to payload'
+        payload:
+          type: object
+          description: 'Payload of the reaction added event'
+          properties:
+            token:
+              type: string
+            team_id:
+              type: string
+            event:
+              type: object
+              properties:
+                user:
+                  type: string
+                  description: User ID who performed this event
+                reaction:
+                  type: string
+                  description: The only reaction that you need is a heart emoji
+                item_user:
+                  type: string
+                  description: |
+                    User ID that created the original item that has been reacted
+                    to
+                item:
+                  type: object
+                  properties:
+                    channel:
+                      type: string
+                      description: Channel information of original message
+                    ts:
+                      type: string
+                      description: Timestamp information of original message
+                event_ts:
+                  type: string
+                  description: Reaction timestamp
+        type:
+          type: string
+        accepts_response_payload:
+          type: boolean `}
+</CodeBlock>
+
+Additionally, you will also be adding the `acknowledge` schema that makes use of the `envelope_id` attribute to send a reply back to Slack acknowledging that event has been received.
+
+<CodeBlock language="yaml">
+{`    acknowledge:
+      type: object
+      properties:
+        envelope_id:
+          type: string
+          description: 'Unique ID of acknowledged payload'
+        payload:
+          type: object
+          description: 'Optional payload of event' `}
+</CodeBlock>
+
+## Define channels 
```

Comment:
this is a continuation right? so I guess instead of define channels it should say - add message to channel? or something similar